docs/en/Running-Presto-with-Alluxio.md

```diff
@@ -9,52 +9,95 @@ priority: 2
 * Table of Contents
 {:toc}
 
-This guide describes how to run [Presto](https://prestodb.io/) with Alluxio, so
-that you can easily use Presto to query Hive tables stored in Alluxio's tiered storage.
+## Overview
+
+[Presto](https://prestodb.io/)
+is an open source distributed SQL query engine for running interactive analytic queries
+on data at a large scale.
+This guide describes how to run Presto to query Alluxio as a co-located distributed cache for
+Hive tables, where the data source can be AWS S3, Azure blob store, HDFS and many others.
+As a result, as long as a block is cached on one Alluxio worker node,
+it can also benefit other Presto nodes querying the same data.
 
 ## Prerequisites
 
-The prerequisite for this part is that you have [Java](Java-Setup.html). And the Java version must use Java 8 Update 60 or higher (8u60+), 64-bit.
-Alluxio cluster should also be set up in accordance to these guides for either
+* [Setup Java](Java-Setup.html) for Java 8 Update 60 or higher (8u60+), 64-bit.
+* [Deploy Presto](https://prestodb.io/docs/current/installation/deployment.html).
+This doc assumes and is tested with `presto-0.191`.
+* Alluxio cluster should also be set up according to these guides for either
 [Local Mode](Running-Alluxio-Locally.html) or [Cluster Mode](Running-Alluxio-on-a-Cluster.html).
+* Make sure that the Alluxio client jar is available.
+  This Alluxio client jar file can be found at `{{site.ALLUXIO_CLIENT_JAR_PATH}}` in the tarball
+  downloaded from Alluxio [download page](http://www.alluxio.org/download).
+  Alternatively, advanced users can compile this client jar from the source code
```

Comment:
Talking about needing to compile from source makes integration with Presto seem less stable. Do you think it will be common for users to need to compile from source?

Reply:
Agree. Removed